sktime/forecasting/arch.py

```diff
@@ -0,0 +1,307 @@
+#!/usr/bin/env python3 -u
+# -*- coding: utf-8 -*-
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+
+"""Implements Generalized Autoregressive Conditional Heteroskedasticity (GARCH) models and it's variants."""
+
+__author__ = ["Vasudeva-bit"]
+
+from sktime.forecasting.base import BaseForecaster
+from sktime.utils.validation._dependencies import _check_soft_dependencies
+
+_check_soft_dependencies("arch", severity="warning")
+
+class ARCH(BaseForecaster):
+    """Generalized Autoregressive Conditional Heteroskedasticity (GARCH) model for forecasting 
+    votility in high frequency time series data.
+
```

Comment:
should say explicitly which package and model therein it interfaces

Reply:
Oh okay, understood.

Reply:
I would say "Directly interfaces ARCH models from python package `arch`"

Reply:
Okay, your option is more clear so will update it as "Directly interfaces ARCH models from python package `arch`"